nautobot/core/templates/inc/nav_menu.html

```diff
@@ -21,6 +21,52 @@
 
     <ul class="list-unstyled my-0 overflow-auto pt-10 w-100">
         {% if request.user.is_authenticated %}
+            {% if nav_menu_version_control %}
+                <li>
+                    {% url 'plugins:nautobot_version_control:branch_list' as branch_list_url %}
+                    <a class="nb-sidenav-list-item nb-sidenav-list-item-flat{% if request.path == branch_list_url %} nb-sidenav-list-item-active{% endif %}" href="{{ branch_list_url }}">
+                        <img alt="list-unordered icon" src="{% static 'nautobot-icons/list-unordered.svg' %}">
+                        <span>Branches</span>
+                    </a>
+                </li>
+                <li>
+                    <form data-bs-theme="dark" id="sidenav-branch-picker">
+                        <img alt="branch icon" src="{% static 'nautobot-icons/branch.svg' %}">
+                        <select
+                            aria-label="Find a branch..."
+                            class="form-select nautobot-select2-api"
+                            data-allow-clear="false"
+                            data-url="{% url 'plugins-api:nautobot_version_control-api:branch_list' %}"
```

Comment:
@norbert-mieczkowski-codilime This is throwing an error for me in latest VC build, I think it needs to be corrected to `plugins-api:nautobot_version_control-api:branch-list` here (note `branch-list` rather than `branch_list`), because whoever said we were consistent between our UI patterns and our API patterns?